### PR TITLE
lapack: add set(i,j,value)

### PR DESCRIPTION
**Pull request by davydden**

### CODE REVIEW

File: include/deal.II/lac/lapack_full_matrix.h [hunk 1]
```diff
@@ -135,6 +135,15 @@ class LAPACKFullMatrix : public TransposeTable<number>
   LAPACKFullMatrix<number> &
   operator/= (const number factor);
 
+  /**
+   * Set a particular entry of the matrix to a value.
```
Comment (jppelteret) [inline include/deal.II/lac/lapack_full_matrix.h]: `@p value`

File: include/deal.II/lac/lapack_full_matrix.h [hunk 2]
```diff
@@ -135,6 +135,15 @@ class LAPACKFullMatrix : public TransposeTable<number>
...
+   * Thus, calling <code>A.set(1,2,3.141);</code> is entirely equivalent to the operation <code>A(1,2) = 3.141;</code>.
+   * This function exists for compatibility with the various sparse matrix objects.
+   */
+  void set (const size_type   i,
+            const size_type   j,
+            const number  value);
```
Comment (jppelteret) [inline include/deal.II/lac/lapack_full_matrix.h]: Please fix the indentation of the parameters, both here and in the function definition.

**Comment by davydden** [comment]
@jppelteret fixed.

**Comment by bangerth** [comment]
OK if (i) you add a test and (ii) a changelog entry.

**Comment by davydden** [comment]
> OK if (i) you add a test and (ii) a changelog entry.

done.

File: tests/lapack/full_matrix_35.cc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  AssertThrow(A(0,1) == 2., ExcInternalError());
+
+  deallog << "OK" << std::endl;
+}
+
+int main()
+{
+  const std::string logname = "output";
+  std::ofstream logfile(logname.c_str());
+  logfile.precision(3);
```
Comment (masterleinad) [inline tests/lapack/full_matrix_35.cc]: Would you mind to use `initlog`?

Comment (davydden) [reply tests/lapack/full_matrix_35.cc]: done.

**Pull request merged**